src/gennylib/src/context.cpp

```diff
@@ -78,6 +78,13 @@ WorkloadContext::WorkloadContext(const Node& node,
 
     _registry = genny::metrics::Registry(std::move(format), std::move(metricsPath));
 
+    auto& ignoredMetrics = (*this)["IgnoredMetrics"];
```

Comment:
Consider using `getPlural("IgnoredMetric","IgnoredMetrics")` (info [here](https://github.com/mongodb/genny/blob/4e6e1c7da4d830f44fbf4a65c4d4e3bae2011a6a/src/gennylib/include/gennylib/Node.hpp#L501) to allow either config syntax. (I bring this up just to show off syntax features you might not be familiar with, I don't think it matters _that_ much in the long run.)